docs/team/xiongjya.md

```diff
@@ -14,35 +14,52 @@ and it still has the benefits of a Graphical User Interface (GUI) created with J
 
 Given below are my contributions to the project.
 
-* **Code contributed**: Around 4.5 kLoC contribution to the project.
+* **Code contributed**: Around 5 kLoC contribution to the project. [Reposense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=f13&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=xiongjya&tabRepo=AY2122S1-CS2103T-F13-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false)
+
+
+
+* **New Feature**: Added the ability to add remarks to elderly contacts.
+    * What it does: Allows the user to add additional notes in the form of a remark, to each elderly contact that has been added into NurseyBook.
+    * Justification: Instead of using other applications/pen or paper to take down additional notes for each elderly, the user can keep track of such information within the application itself. There will be no need for the user to switch between different applications.
+
+
+* **New Feature**: Sort newly added tasks into NurseyBook in chronological order.
+    * What it does: When the user adds a task into NurseyBook, the task will not be added to the end of the list. Instead, it will be slotted into a position in the task list, to make sure that the task list displays tasks in a chronological order.
```

Comment:
How about "when a user adds a task into NurseyBook, the task is positioned in the list according to its time such that the tasks in the list are always chronological"?

Reply:
Just a suggestion, current wording is fine as well!

Reply:
Alright! Such wording seems clearer too.